packages/content-utils/src/runtime/git.ts

```diff
@@ -92,6 +92,19 @@ export async function listGitTrackedFiles(): Promise<string[]> {
 	return files;
 }
 
+function getRepoRoot(): string {
+	const result = spawnSync('git', ['rev-parse', '--show-toplevel'], {
+		cwd: contentPath,
+		encoding: 'utf-8',
+	});
+
+	if (result.error) {
```

Comment:
What could cause an error?

Reply:
Not having Git installed, creating a symlink cycle and other weird stuff on the system.